deployment/src/main/java/io/quarkiverse/quinoa/deployment/packagemanager/FrameworkType.java

```diff
@@ -26,6 +26,7 @@
 public enum FrameworkType {
 
     REACT("build", "start", 3000, Set.of("react-scripts", "react-app-rewired", "craco")),
+    VUE_LEGACY("dist", "start", 3000, Set.of("vue-cli-service")),
```

Comment:
```suggestion
    VUE_LEGACY("dist", "serve", 3000, Set.of("vue-cli-service")),
```

Reply:
I think here should be `serve`

Reply:
I just followed what is in the Start script of the integration test package.json?

Reply:
@melloware in the package.json, I might have changed it be compatible with the previous way. Create a new Vue app to be sure